### DISCUSSION TITLE
[MDN Community] Call for International Language Discord Moderators

### DISCUSSION THREAD

**Discussion by zfox23** [mdn/mdn-community#460] [Localization] [answered]
Hi y'all!

We at MDN are trying to more authentically connect with our community in real time using [our Discord community](https://discord.gg/S2TnyQvaTJ). I've been making some changes over there to make it easier to use, and I've run into an important challenge.

We currently have _active_ (yay!) #korean and #spanish language channels on the MDN Discord with no official volunteer moderators. The MDN Content team doesn't have native Korean or Spanish language speakers, and we need help making sure the content of those channels follows our community guidelines.

Is anyone who speaks Korean or Spanish willing to moderate one of those channels? If you're interested in taking on this responsibility and getting a wicked cool Discord role, you can reply to this discussion thread, let us know in [our #general Discord channel](https://discord.gg/p3F6MWnPA8), or DM me @ZachAtMozilla. Thank you!

CC: @mdn/localization-team-leads

**Comment by hochan222** [comment, marked as answer]
I replied to the Discord thread. Thank you :) 

https://discord.com/channels/1009925603572600863/1171105108646445076/1172025418803581029

**Comment by pransh15** [comment]
Closing this as it was answered on Discord.
